src/ffishjs.cpp

```diff
@@ -175,6 +175,11 @@ class Board {
     return this->pos.fen();
   }
 
+  std::string fen(bool showPromoted, int countStarted) const {
+    countStarted = std::min<unsigned int>(countStarted, INT_MAX); // pseudo-unsigned
```

Comment:
I checked the source of _Fairy-SF_ and I think you can safely delete this line.

The method `counting_ply()` already applies all required checks:
https://github.com/ianfab/Fairy-Stockfish/blob/2f49208dd18124e1126454a8fbcf6ea997cd3495/src/position.h#L1178-L1180

Moreover, applying the min-operator by comparing with `INT_MAX` is not necessary, unless I'm missing something.